src/libutil/ustring.cpp

```diff
@@ -637,11 +596,12 @@ ustring::getstats(bool verbose)
 size_t
 ustring::hash_collisions(std::vector<ustring>* collisions)
 {
-    std::lock_guard<std::mutex> lock(collision_mutex);
-    if (collisions)
-        for (const auto& c : all_hash_collisions)
-            collisions->emplace_back(ustring::from_unique(c.first));
-    return all_hash_collisions.size();
+    if (collisions) {
+        // Disabled for now
```

Comment:
You don't need to keep a collection of all hash collisions if you put the pointer into the "hashed" member on collision...then you can just iterate through all the strings in the system, and print out those who's hash is actually a pointer.